tests/integration/test_main.py

```diff
@@ -44,7 +40,8 @@ def test_integrate(juju: jubilant.Juju):
 def test_integrate_post_start(juju: jubilant.Juju):
     juju.deploy(microovn_charm_path)
     juju.deploy(token_distributor_charm_path)
-    juju.wait(jubilant.all_active)
+    juju.wait(jubilant.all_active,"microcluster-token-distributor")
+    juju.wait(jubilant.all_maintenance,"microovn")
```

Comment:
You likely need something like this instead:
https://github.com/canonical/jubilant/blob/3d20fa4bcf499b720628c046e35361a5e3e6c7f4/docs/tutorial/getting-started.md?plain=1#L82